lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -255,13 +256,13 @@ <h3 class="error-msgs" id="error-msg"></h3>
          }
          if($('#invoice_number').val() == "") {
                      $('#error-msg').attr('class','error-msgs')
-                     $('#error-msg').html("${_('The Invoice Number field cannot be empty.')}").show();
+                     $('#error-msg').html("${_('The Invoice Number field cannot be empty.') | n, h, js_escaped_string}").show();
```

Comment:
@douglashall You don't need to make any changes here.  This is just an FYI as I thought more about this case.

It is possible that the linter might complain about this in the future and that we should instead perform the escaping in JavaScript.  I'm not sure if the `show()` method is only available in on the results of `html()`, and not `text()`?
I'm not sure if any of the following would work:
```
$('#error-msg').text("${_('The Invoice Number field cannot be empty.') | n, js_escaped_string}").show();
```
```
$('#error-msg').text("${_('The Invoice Number field cannot be empty.') | n, js_escaped_string}").html().show();
```
```
$('#error-msg').text("${_('The Invoice Number field cannot be empty.') | n, js_escaped_string}")
$('#error-msg').html().show();
```
Alternatively, you could make use of `HtmlUtils.setHtml()`.